src/Blueman/Console/Command/ConvertCommand.php

```diff
@@ -65,10 +81,23 @@ protected function execute(InputInterface $input, OutputInterface $output)
             );
         }
 
+        $tests = array();
+        if('true' === $input->getOption('tests-include')) {
```

Comment:
Can you swap the order to match the coding style of the rest of the code? e.g. `if ($input->getOption('tests-include') === true) {`